src/frontend/components/Sidebar/Sidebar.js

```diff
@@ -6,28 +7,44 @@ export const Sidebar = () => {
             <div>
                 <ul className="text-sm side-list">
                     <li>
-                        <span className='material-icons'>home</span>
-                        <a className="links">Home</a>
+                        <Link to="/" className="links">
+                            <span className='material-icons'>home</span>
+                            <span className="side-list-title">Home</span>
+                        </Link>
+
                     </li>
                     <li>
-                        <span className='material-icons'>explore</span>
-                        <a className="links">Explore</a>
+                        <NavLink to="/videos" className={({ isActive }) => isActive ? "active links" : "links"}>
```

Comment:
You can extract this isActive inside a function so that you don't repeat it

Reply:
This is a react router 6 way of adding active state to `Navlink`.
Adding link to official docs for reference https://reactrouter.com/docs/en/v6/getting-started/tutorial#active-links